CHANGELOG.md

```diff
@@ -1,6 +1,29 @@
-## 3.2.0 (Unreleased)
-
-Changes in progress for v3.2.0 are available at [.changes/v3.2.0](https://github.com/vmware/terraform-provider-vcd/tree/master/.changes/v3.2.0) until the release.
+## 3.2.0 (March 10, 2021)
```

Comment:
Given that we don't know when HashiCorp will actually release:

```suggestion
## 3.2.0 (March 11, 2021)
```

Reply:
Changed.